tests/unit/tls_test.cc

```diff
@@ -170,6 +170,9 @@ SEASTAR_TEST_CASE(test_x509_client_with_system_trust_and_priority_strings) {
         "ECDHE-ECDSA-AES256-GCM-SHA384",
         "AES128-GCM-SHA256"
     });
+    auto set_priority_string = [](tls::credentials_builder & b, const sstring & prio) {
+        b.set_cipher_string(prio);
```

Comment:
Is it worth testing `TLSv1.3`?

Reply:
I can add that sure

Reply:
Oh that's actually why I think the other test was failing because it would go to TLSv1.3...

Reply:
I think it's worth adding.